soc/arm/st_stm32/stm32l4/power.c

```diff
@@ -30,7 +30,11 @@ void sys_set_power_state(enum power_states state)
 		LL_DBGMCU_EnableDBGStopMode();
 #endif /* CONFIG_DEBUG */
 		/* ensure HSI is the wake-up system clock */
```

Comment:
Please update comment

Reply:
Comment updated in latest modified branch, thanks.